modules/indexExchangeBidAdapter.js

```diff
@@ -419,6 +426,9 @@ var IndexExchangeAdapter = function IndexExchangeAdapter() {
     _IndexRequestData.targetAggregate = {'open': {}, 'private': {}};
 
     if (!utils.hasValidBidRequest(bidArr[0].params, requiredParams, ADAPTER_NAME)) {
+      for (var i = 0; i < bidArr.length; i++) {
+        passOnBid(bidArr[i].placementCode);
+      }
```

Comment:
@arvending the bids array contains multiple bids across different adUnits though.  Also, that bid order is pretty much arbitrary (I think it ends up being definition order or something), so why you would want to return NO_BIDs on _all_ bids across _all_ adUnits involved in an auction because an arbitrary bid invalid makes no sense.  Maybe I'm missing something with the indexExchange exception that changes things?

@mkendall07 what do you think about this?